index.html

```diff
@@ -186,7 +186,7 @@ <h1 id="title">
         <script type="module">
           const host = document.getElementById('host');
           const shadowRoot = host.attachShadow({ mode: 'open' });
-          shadowRoot.innerHTML = `<section class="demo-item">
+          shadowRoot.innerHTML = `<link rel="stylesheet" href="/css/prism.css" /> <section class="demo-item">
```

Comment:
@jgerigmeyer Can you tell why prism styles aren't applying in the code examples here?